src/main/java/com/github/cameltooling/lsp/internal/parser/CamelKModelineParser.java

```diff
@@ -16,10 +16,12 @@
  */
 package com.github.cameltooling.lsp.internal.parser;
 
+import org.eclipse.lsp4j.Position;
 import org.eclipse.lsp4j.TextDocumentItem;
 
 public class CamelKModelineParser {
 
+	//Unify this and Insertion definitions?
```

Comment:
better to create an issue than a comment in the code.
Also what is the Insertion Definitions?